<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Game</class>
 <widget class="QDialog" name="Game">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>918</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Labyrinth 2015 - Game</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>551</width>
     <height>67</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="player1_label">
        <property name="text">
         <string>Player 1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="player1Name_label">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="player1Score">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="player2_label">
        <property name="text">
         <string>Player 2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="player2Name_label">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="player2Score">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="player3_label">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Player 3</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="player3Name_label">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="player3Score">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="player4_label">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Player 4</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="player4Name_label">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="player4Score">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>150</y>
     <width>671</width>
     <height>531</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout"/>
  </widget>
  <widget class="QLabel" name="infoLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>700</y>
     <width>621</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Some information text here</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>340</y>
     <width>101</width>
     <height>62</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QPushButton" name="saveButton">
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="undoButton">
      <property name="text">
       <string>Undo</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
